schemas/resource-schema.json

```diff
@@ -191,6 +196,22 @@
               "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
             }
           }
+        },
+        "_dd": {
+          "type": "object",
+          "description": "Internal properties",
+          "properties": {
+            "span_id": {
+              "type": "string",
```

Comment:
just checked, APM defines these as unit64 in their protobuf files: https://github.com/DataDog/dd-go/blob/prod/pb/proto/trace/agent/model/span.proto#L13-L14

does the "integer" type handle 64bit? if so, we can use it instead of string

Reply:
This will work on iOS 👌 , as we use `Int64` for all numeric / integer values in RUM models.

Reply:
This'll work on our side too

Reply:
Actually, 64 bit integer is not supported on the JS side 😞